tests/system/projects/test_project.py

```diff
@@ -485,6 +485,25 @@ def test_local_pipeline(self):
     def test_kfp_pipeline(self):
         self._test_new_pipeline("kfppipe", engine="kfp")
 
+    def test_kfp_pipeline_with_resource_param_passed(self):
+        project_name = "test-pipeline-with-resource-param"
+        self.custom_project_names_to_delete.append(project_name)
+        project = mlrun.new_project(project_name, context=str(self.assets_path))
+
+        code_path = str(self.assets_path / "sleep.py")
+        workflow_path = str(self.assets_path / "pipeline_with_resource_param.py")
+
+        project.set_function(
+            name="func-1",
+            func=code_path,
+            kind="job",
+            image="mlrun/mlrun",
+            handler="handler",
+        )
+        # set and run a two-step workflow in the project
+        project.set_workflow("paramflow", workflow_path)
+        project.run("paramflow", engine="kfp", watch=True)
```

Comment:
I thought you would be passing a memory param and verify that it was passed correctly.
Did I misunderstand the feature?

Reply:
it had default value which the pipeline used, but took your comment and added explicit parameter